unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -38,19 +39,23 @@ export class CreatorLock extends React.Component {
       <LockIconBar lock={lock} toggleCode={this.toggleEmbedCode} />
     </LockIconBarContainer>)
 
+    let status = 'Not found'
```

Comment:
hum. What is this? It looks like if there is no transaction you would mark the lock as not found when this should not be the case. We can have cases where the app would now know about the transaction fromn which this lock is from and still the lock would be "found" ;)